src/components/Footer.astro

```diff
@@ -0,0 +1,3 @@
+<div class="flex h-20 items-center justify-between px-5 py-3 md:px-10">
+  <p>© Copyright {new Date().getFullYear()} DSSD Madison</p>
```

Comment:
We're definitely not actually going to register for copyright on this website so I think it's illegal for us to put this. Maybe we could just add some more redundancy of contact links in the footer?